src/NuGet.Status/NuGet.Status.csproj

```diff
@@ -199,13 +199,13 @@
       <Version>13.0.1</Version>
     </PackageReference>
     <PackageReference Include="NuGet.Services.Configuration">
-      <Version>2.106.0</Version>
+      <Version>2.111.0</Version>
     </PackageReference>
     <PackageReference Include="NuGet.Services.Status.Table">
-      <Version>2.106.0</Version>
+      <Version>2.111.0</Version>
     </PackageReference>
     <PackageReference Include="NuGet.Versioning">
-      <Version>6.2.1</Version>
```

Comment:
https://www.nuget.org/packages/NuGet.Versioning/6.2.1 is unlisted, just in case update to latest available one, there shouldn't be breaking change, that package doesn't change much..

Reply:
I recommend matching whatever client SDK version Gallery uses.

Reply:
There is no direct dependency on this package, but there's one transitive one, so let's match to that one.
![image](https://github.com/NuGet/NuGet.Services.Status/assets/8766776/c3bb65d2-4e96-44a7-b4fe-cff984e28218)

Reply:
Seems fine. If you want to remove the direct dependency and leave it transitive that might be a nice optional clean-up. Or just match the version as you described.

Reply:
I matched with Gallery version.